File: hashmap/ransom_note/solution_test.go
package ransom_note

import "testing"

func TestPossibleRansomNote(t *testing.T) {
	actual := canConstruct("aa", "aab")
	if !actual {
		t.Errorf("Expected %t to be true", actual)
	}
}

func TestImpossibleRansomNote(t *testing.T) {
	actual := canConstruct("aabc", "aa")
	if actual {
		t.Errorf("Expected %t to be true", actual)
	}
}

File: hashmap/ransom_note/solution_test.py
import unittest

from hashmap.ransom_note.solution import Solution


class TestSolution(unittest.TestCase):
    def test_possible_ransom_note(self):
        sol = Solution()
        actual = sol.canConstruct("aa", "aab")
        self.assertTrue(actual)

    def test_impossible_ransom_note(self):
        sol = Solution()
        actual = sol.canConstruct("aac", "aab")
        self.assertFalse(actual)